backend/src/LibParser/FSharpToWrittenTypes.fs

```diff
@@ -302,6 +304,7 @@ module Expr =
     | SynExpr.Const(SynConst.Int32 n, _) -> WT.EInt(id, n)
     | SynExpr.Const(SynConst.Int64 n, _) -> WT.EInt(id, int64 n)
     | SynExpr.Const(SynConst.UInt64 n, _) -> WT.EInt(id, int64 n)
+    | SynExpr.Const(SynConst.Byte n, _) -> WT.EInt8(id, int8 n)
```

Comment:
`int8` and bytes aren't the same. Bytes are the same as `uint8` (that is, the range is 0 to 255, not -128 to 127)

Reply:
I have this uncommitted change localy  do you expect doing it differently?
```
    | SynExpr.Const(SynConst.SByte n, _) -> WT.EInt8(id, int8 n)
    | SynExpr.Const(SynConst.Byte n, _) -> WT.EInt8(id, int8 n) 
```